### PR TITLE
Add CreateCollapse 2/c

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Adds the custom Collapse for the create flow

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a> 
    * **PR #1971** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971?utm_source=stack-comment).

File: src/styles/antd-overrides/collapse.scss [hunk 1]
```diff
@@ -169,3 +169,16 @@
...
+// Create flow collapse styling
+.create-collapse {
+  .ant-collapse-header {
+    padding-left: 0 !important;
+    border-bottom: solid 1px var(--stroke-tertiary);
+  }
+
+  .ant-collapse-content-box {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
```
Comment (tomquirk) [inline src/styles/antd-overrides/collapse.scss:182]: **question**: any way we can avoid `!important`? Probably greater specificity will do it.

**Review by tomquirk — APPROVED** [review]
preapproving!

File: src/styles/antd-overrides/collapse.scss (hunk 1 above)
Comment (wraeth-eth) [reply src/styles/antd-overrides/collapse.scss:182]: It is probable - I'm a noob at this stuff so I just found I had to add `!important` to make it work. Let me know a better way 🙏

**Pull request merged**